docs/admin/install/docker.rst

```diff
@@ -1114,6 +1127,14 @@ Example SSL configuration:
 
     .. seealso:: :setting:`django:EMAIL_HOST_PASSWORD`
 
+.. envvar:: WEBLATE_EMAIL_HOST_PASSWORD_FILE
+
+    Path to file containing the e-mail authentication password.
```

Comment:
Please do a small grammatic correction to:

*Path to the file containing the e-mail authentication password.*

Reply:
Thanks for the implementation!